bundles/org.openhab.binding.boschshc/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.boschshc</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: BoschSHC Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
+      <version>1.54</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk15on</artifactId>
+      <version>1.54</version>
+      <scope>compile</scope>
+    </dependency>
+  </dependencies>
```

Comment:
@GerdZanker Could you add an info about it in the NOTICE file? And is it really necessary that we include both org.bountycastle dependencies?

Reply:
I think it should be sufficient to include one.

Reply:
I think it is necessary to include both as compile dependencies since the intent is for those jars to be shaded into the binding jar. Transient dependencies are not automatically shaded alongside a compile dependency so they have to be manually specified. If you exclude one of those dependencies then you will be forced to include the missing dependency in the feature.xml instead. Since I don't see that these dependencies are used by any other bindings then you should include them as compile dependencies.

Reply:
I meant the occurrence in NOTICE. I'm not sure if @coeing meant the pom.xml or NOTICE.

Reply:
Well yes, I do agree that these should added to the NOTICE. Just to be safe, include both dependencies in the NOTICE file.

Reply:
Thanks for the info to include both dependencies. Is there a documentation which information of the dependencies should go into the NOTICE file?

Reply:
>  Is there a documentation which information of the dependencies should go into the NOTICE file?

Not that I'm aware of, but you can look at other bindings as an example.